recipes/pango/all/conanfile.py

```diff
@@ -38,16 +38,28 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
+        if(self.options.xft == "auto"):
+            self.options.xft = (self.settings.os == 'Linux' or self.settings.os == 'FreeBSD')
+        if(self.options.fontconfig == "auto"):
+            self.options.fontconfig = (self.settings.os != 'Windows' and self.settings.os != 'Macos')
+
+        if self.options.xft and self.settings.os != 'Linux' and self.settings.os != 'FreeBSD':
+            raise ConanInvalidConfiguration('Xft can only be used on Linux and FreeBSD')
+
     def build_requirements(self):
         self.build_requires("pkgconf/1.7.3")
         self.build_requires("meson/0.54.2")
 
     def requirements(self):
-        if self.settings.os != "Windows":
+        if self.options.freetype:
+            self.requires("freetype/2.10.4")
+
+        if self.options.fontconfig:
             self.requires("fontconfig/2.13.92")
-        if self.settings.os == "Linux":
+        if self.options.xft:
```

Comment:
Should this option be deleted on Windows/Mac?

Reply:
Possibly, currently we throw a `ConanInvalidConfiguration` if Xft is set to `True` on Windows or Mac. Not sure what's more sensible.

Reply:
Let's see if others comment